src/main/java/de/medizininformatikinitiative/fhir_data_evaluator/populations/MeasurePopulation.java

```diff
@@ -21,12 +21,13 @@ public record MeasurePopulation(int count) {
     public static MeasurePopulation ZERO = new MeasurePopulation(0);
     public static MeasurePopulation ONE = new MeasurePopulation(1);
 
-    public MeasurePopulation merge(MeasurePopulation other) {
-        return new MeasurePopulation(count + other.count);
-    }
-
-    public MeasurePopulation deepCopy() {
-        return new MeasurePopulation(count);
+    /**
+     * Increments the count of the measure population.
+     *
+     * @param increment whether the count of the measure population should be incremented
+     */
+    public MeasurePopulation increment(boolean increment) {

```

Comment:
should not take an increment boolean. instead just don't call it if you don't like to increment